weave-and-docker-platform/3-machine-weaveproxy-swarm.md

```diff
@@ -41,91 +39,137 @@ this setup with few simple shell scripts.
+On weave-1 run: 
 
-Then we can connect the cluster with `weave connect $(docker-machine ip weave-1)` for `weave-2` and `weave-3`.
+~~~bash
+weave launch
+~~~
 
-As we shown in the two previous chapters, we can run these commands remotely with the help of `DOCKER_CLIENT_ARGS`.
+Set the weave env for `weave-1`
 
-You can find a working script that implements this in [`scripts/2-weave-launch.sh`][step2].
+~~~bash
+eval "$(weave env)"
+~~~
+
+Next launch weave on `weave-2` with --init-peer-count set to 3:
+
+~~~bash
+weave launch-router --init-peer-count 3
+~~~
+
+Next, specify the TLS settings if asked: 
+
+~~~bash
+weave launch-proxy --tls \
+         --tlscacert /var/lib/boot2docker/ca.pem \
+         --tlscert /var/lib/boot2docker/server.pem \
+         --tlskey /var/lib/boot2docker/server-key.pem
+~~~
+
+and then set the weave environment variable for and `weave-2`
+
+~~~bash
+eval "$(weave env)"
+~~~
+
+Next on `weave-2` connect the cluster to our bootstrap node, `weave-1`: 
+
+~~~bash
+weave connect $(docker-machine ip weave-1)
+~~~
+
+check to see that all went well: 
+
+~~~bash
+weave status
+~~~
+
+Follow the same steps for `weave-3` as you did for `weave-2` above. 
+
+
+  >>A useful script which launches weave and sets up the hosts and connects the cluster can be found here: [`scripts/2-weave-launch.sh`][step2].
 
-<div class="alert alert-warning">
-<b>Please note</b> that in Weave there is no notion of master/slave or any other roles of the nodes. Here we simply
-picked <code>weave-1</code> as a sort of bootstrap node, but we could also pass all IPs or DNS names to <code>weave launch</code>
-and thereby also avoid having to set <code>-initpeercount</code> explicitly. However, with Docker Machine on VirtualBox
-we do not know the IP addresses and don't have DNS. You should be able to use DNS with one of the cloud drivers, such as
-Microsoft Azure or Google Compute Engine.
-</div>
 
 ### Setting up Swarm agents against Weave proxy
 
-This step is a little bit more involved and I will refrain from covering it in detail, as this is really a work-around to
-an [issue in Docker Machine](https://github.com/docker/machine/issues/1334). In essence, currently we have to restart
-Swarm agents with a new discovery token and make them register TCP port 12375 for Weave proxy. You can find out how it's
-done in [`scripts/3-replace-swarm-agents.sh`][step3].
+This step is more involved, and is a work-around to reported Docker Machine issue, which we will refrain from covering in detail. For more information, refer to [issue in Docker Machine](https://github.com/docker/machine/issues/1334). In short, swarm agents are restarted with a new discovery token and registered to TCP port 12375 for Weave Proxy. More information can be found in the following script [`scripts/3-replace-swarm-agents.sh`][step3].
 
 ### Put it all together
 
-First you shoild obtain the repository with
+First clone the `Guides` repository:
```

Comment:
I think this should either say `weaveworks/guides` or just "guides", in any case it'd be to make either a hyperlink to repo URL.